noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -680,10 +1185,15 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST5">
         <title>Series status</title>
+        <description>
+            Tests whether a non-finalized Series exists in the archive.
```

Comment:
`Tests whether a non-finalized series exists in the archive. A series is considered non-finalized if a finalized date or a finalizing party is not specified, or if the series status is not 'Avsluttet periode'`